readme.md

```diff
@@ -3,7 +3,9 @@ To get started:
 1. clone this repository
 2. `yarn install`
 3. `gulp dev`
+4. `gulp ship`
 
 Edit files in the /src folder! The `gulp dev` command will build those files and output them to the /dis folder. Gulp watches for file changes, and anytime a /src file is changed it is build and output to /dis.
+The `gulp ship` command build files and outputs to the /dis folder without launching watcher.
```

Comment:
After changing gulpfile, remember to change these to "deploy"

Reply:
Done.